.gitignore

```diff
@@ -1,42 +1,21 @@
-*.rbc
-capybara-*.html
-.rspec
-/log
-/tmp
-/db/*.sqlite3
-/db/*.sqlite3-journal
-/public/system
```

Comment:
@rakvium it was made by rails ) but it wasn't removed, it was added in the end of file